use-cases/using-Intel-QAT-and-kata.md

```diff
@@ -0,0 +1,484 @@
+virtual functions. Using the Intel QAT as an example, the physical device ID 
+is `37c8` and virtual function device ID is `37c9`. The following command checks 
+if VF's are enabled for any of the currently known Intel QAT device ID's. The
+following `ls` command should show the 16 vf's bound to vfio-pci.
+
+```
+$ for i in 0442 0443 37c9 19e3; do lspci -d 8086:$i; done
+
+# Check if vf's are bound to vfio-pci
+$ ls -la /sys/bus/pci/drivers/vfio-pci
+```
+
+## Prepare Kata Containers
+
+### Download Kata Kernel Source
+
+This follows the instructions from 
+[https://github.com/kata-containers/packaging/tree/master/kernel](https://
+github.com/kata-containers/packaging/tree/master/kernel). The config for the 
+kata-kernel is at 
+[https://github.com/kata-containers/packaging/tree/master/kernel/configs](
+https://github.com/kata-containers/packaging/tree/master/kernel/configs) In 
+this example, the latest kernel supported by Kata is automatically used.
+There are some patches that must be installed as well which the 
+build-kernel.sh script should automatically apply, but if you are using a 
+different kernel version then they may need to be manually applied. Since the 
+Kata Containers kernel has a minimal set of kernel flags set, the config must 
+be updated to set some of the `CRYPTO` and `STACKPROTECTOR` flags to enabled. 
+This may change with different kernel versions. The following was tested with 
+kernel `v4.19.28-41`.
+
+```sh
+$ mkdir -p $GOPATH
+$ cd $GOPATH
+$ go get -v github.com/kata-containers/packaging
+$ $GOPATH/src/github.com/kata-containers/packaging/kernel/build-kernel.sh setup
+```
+
+### Build Kata Kernel
+
+```sh
+$ export LINUX_VER=$(ls -d kata*)
+$ sed -i 's/EXTRAVERSION =/EXTRAVERSION = .qat.container/' $LINUX_VER/Makefile
+
+# Change kernel config options to support QAT
+$ sed -i 's/# CONFIG_PCI_IOV is not set/CONFIG_PCI_IOV=Y/' $LINUX_VER/.config
+$ sed -i 's/# CONFIG_CRYPTO_SHA1 is not set/CONFIG_CRYPTO_SHA1=Y/' $LINUX_VER/.config
+$ sed -i 's/# CONFIG_CRYPTO_DH is not set/CONFIG_CRYPTO_DH=Y/' $LINUX_VER/.config
+$ sed -i 's/# CONFIG_STACKPROTECTOR is not set/CONFIG_STACKPROTECTOR=Y/' $LINUX_VER/.config
+$ $GOPATH/src/github.com/kata-containers/packaging/kernel/build-kernel.sh build
+```
+
+> **Note: There may be additional kernel parameters that need to be set to yes 
+> when compiling.**
+
+### Copy Kata Kernel
+
+```sh
+$ mkdir -p $KATA_KERNEL_LOCATION
+$ cp $LINUX_VER/arch/x86/boot/bzImage $KATA_KERNEL_LOCATION/vmlinuz-${LINUX_VER}_qat
+```
+
+### Prepare Kata root filesystem
+
+These instructions build upon the OS builder instructions located in the  
+[Developer Guide](../Developer-Guide.md). This example will use Clear Linux 
+(Kata Containers default) as the root filesystem with systemd as the init and 
+will add in the `kmod` binary which is not a standard binary in a Kata image.  
+Docker should already be installed. If a custom kata-agent needs to be used 
```

Comment:
Can you change this to `Docker should  already be installed on your system` ? Reading the previous line made me think you were referring to the rootfs.

Reply:
Will update